### ISSUE TITLE
False Positive with vi.defineHelper and no-standalone-expect

### ISSUE THREAD

**Issue by leeyi45** [vitest-dev/eslint-plugin-vitest#893]
`vi.defineHelper` allows us to create functions within which `expect` assertions can reside:

```ts
const helper = vi.defineHelper((x: number) => {
  expect(x).toEqual(0);
})
```

`no-standalone-expect` highlights this as a violation, when it probably should be allowed. Though one can configure the function to be excluded in the rule's options, I think that helper functions created with `defineHelper` should be excluded altogether.

Perhaps having the helper function highlighted for `no-standalone-expect` instead could be considered a correct violation.
